src/provider/index.ts

```diff
@@ -136,8 +142,9 @@ export class Main {
         const appWindow = this._model.getWindow(identity);
         if (appWindow) {
             appWindow.addIntentListener(payload.intent);
-            return Promise.resolve();
+            return;
         } else {
+            // TODO? [SERVICE-429] Should this be an FDC3Error? Or is the AppWindow model an implementation detail and is OK to throw a non-FDC3 error?
```

Comment:
Agree all TODOs should be treated with equal importance (and addressed sooner rather than later!!). I tend to use `?` and `!` as I'm using [this extension](https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments) which colours them differently. And actually normal `// TODO:`s are coloured more prominently than `?` so should probably use that from now on :)